packunused.cabal

```diff
@@ -79,9 +79,9 @@ executable packunused
   build-depends:
     base                 >=4.5  && <4.10,
     Cabal                >=1.24 && <1.25,
-    optparse-applicative >=0.8  && <0.13,
+    optparse-applicative >=0.8  && <0.14,
     directory            >=1.1  && <1.3,
     filepath             >=1.3  && <1.5,
-    haskell-src-exts     >=1.14 && <1.18,
+    haskell-src-exts     ==1.18.2,
```

Comment:
Oh, thought you meant to fix on a single version. Will amend when I get a minute :+1: